### PR TITLE
Adding specific error messaging and enforced maximum duration to WASM calls

### PR DESCRIPTION
**Pull request by nealerickson-qtm**

### CODE REVIEW

File: tests/integration/test_phir.py
Comment (qciaran) [inline tests/integration/test_phir.py:1]: From the file history, this file was first created in 2023. So far I have been putting the year at which the file was first created/edited in the copyright. However, I believe it is fairly common to include a range of years. We could switch to that format (I might need to double check this is what is done.).

File: python/pecos/foreign_objects/wasmtime.py
Comment (qciaran) [inline python/pecos/foreign_objects/wasmtime.py:1]: LGTM or sorry about that. :D

File: tests/integration/test_phir.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 The PECOS developers
+# Copyright 2024 The PECOS developers
```
Comment (qciaran) [inline tests/integration/test_phir.py:1]:
```suggestion
# Copyright 2023 The PECOS developers
```

**Review by ciaranra — CHANGES_REQUESTED** [review]
LGTM as long as the date for the test_phir.py file is reverted.

**Review by ciaranra — APPROVED** [review]

**Comment by ciaranra** [comment]
LGTM again

**Pull request merged**